src/cclib/parser/utils.py

```diff
@@ -194,6 +195,67 @@ def split(self, line, truncate=True):
         return elements
 
 
+class MoldenReformatter:
+    """Reformat Molden output files."""
+
+    def __init__(self, filestring):
+        self.filestring = filestring
+
+    def scinotation(self, num):
+        """Convert Molden style number formatting to scientific notation.
+        0.9910616900D+02 --> 9.910617e+01
```

Comment:
Are the last two zeros in ```0.9910616900D+02``` significant? If so, you should change the ```%.6e``` to ```%.9e``` below.